tests/CommunityToolkit.Mvvm.UnitTests/Test_Messenger.cs

```diff
@@ -372,6 +374,40 @@ public void Test_Messenger_IRecipient_SomeMessages_NoToken(Type type)
         Assert.IsFalse(messenger.IsRegistered<MessageB>(recipient));
     }
 
+    // See https://github.com/CommunityToolkit/dotnet/issues/198
+    [TestMethod]
+    [DataRow(typeof(StrongReferenceMessenger))]
+    [DataRow(typeof(WeakReferenceMessenger))]
+    public void Test_Messenger_IRecipient_SomeMessages_WithGenericTypeParameters(Type type)
+    {
+        IMessenger? messenger = (IMessenger)Activator.CreateInstance(type)!;
```

Comment:
```suggestion
        IMessenger messenger = (IMessenger)Activator.CreateInstance(type)!;
```

Reply:
Will update all of this in a separate commit 👍